sql_test.go

```diff
@@ -0,0 +1,84 @@
+package sql
+
+import (
+	"context"
+	"io"
+	"testing"
+
+	"github.com/dop251/goja"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modulestest"
+	"go.k6.io/k6/lib"
+	"go.k6.io/k6/lib/testutils"
+	"go.k6.io/k6/metrics"
+)
+
+// Test_SQLLiteIntegration performs an integration test creating an SQLLite database
+func Test_SQLLiteIntegration(t *testing.T) {
+	t.Parallel()
+
+	rt := setupTestEnv(t)
+	_, err := rt.RunString(`
+const db = sql.open("sqlite3", "intg_test.db");
```

Comment:
Let's remove `intg_test.db` in `t.Cleanup()`.

Reply:
It's in `make clean` and `.gitignore,` but I can remove it here as well.